### PR TITLE
doccheck 3: manipulation

### PR DESCRIPTION
**Pull request by banilo**
Next round of documentation tuning...

### CODE REVIEW

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 1]
```diff
@@ -102,30 +110,32 @@ list many files with a "wild-card": \*.nii
...
      :header:
-         informations about the data (slice duration...):
+         low-level informations about the data (slice duration, etc.):
          ``header = img.get_header()``
 
 
-Neuroimaging data can be loaded simply thanks to nibabel_. Once the file is
-downloaded, a single line is needed to load it.
+Neuroimaging data can be loaded in a simple way thanks to nibabel_.
+A nifti file on disk can be loaded by a single line.
```
Comment (AlexandreAbraham) [inline doc/manipulating_visualizing/manipulating_images.rst]: loaded **with** a single line?

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 2]
```diff
@@ -133,14 +143,16 @@ downloaded, a single line is needed to load it.
...
 
-    - a big 4D matrix representing 3D MRI along time, stored in a big 4D
-      NifTi file.
+    - a big 4D matrix representing (3D MRI + 1D for time), stored in a single
+      nifti file.
       `FSL <http://www.fmrib.ox.ac.uk/fsl/>`_ users tend to
       prefer this format.
-    - several 3D matrices representing each volume (time point) of the
-      session, stored in set of 3D Nifti or analyse files.
+    - several 3D matrices representing each time point (single 3D voluem) of the
```
Comment (AlexandreAbraham) [inline doc/manipulating_visualizing/manipulating_images.rst]: typo voluem.

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 3]
```diff
@@ -149,22 +161,22 @@ downloaded, a single line is needed to load it.
...
-  * a file path to a Nifti or Analyse image
-  * any object exposing ``get_data()`` and ``get_affine()`` methods, for
-    instance a ``Nifti1Image`` from nibabel_.
+  * A string variable with a file path to a Nifti or Analyse image
+  * Any object exposing ``get_data()`` and ``get_affine()`` methods, typically
+    a ``Nifti1Image`` from nibabel_.
 
-**Niimg-4D:** Similarly, some functions require 4-dimensional Nifti-like
-data, which we call Niimgs, or Niimg-4D. Accepted inputs are then:
+**Niimg-4D:** Similarly, some functions require sets of Nifti-like
```
Comment (AlexandreAbraham) [inline doc/manipulating_visualizing/manipulating_images.rst]: For me, **sets of Nifti-like** means that we have several which has no sense since we don't know what is the dimension of the nifti-like.

**Comment by AlexandreAbraham** [comment]
Great work, as usual!

File: doc/manipulating_visualizing/manipulating_images.rst (hunk 1 above)
Comment (banilo) [inline doc/manipulating_visualizing/manipulating_images.rst]: Right!

**Comment by banilo** [comment]
:octocat:  says: "Merge me!"

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 4]
```diff
@@ -23,17 +23,20 @@ Loading data
...
 datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
     >>> haxby_files = datasets.fetch_haxby(n_subjects=1)
 
-They return a structure that contains the different file names::
+They return a data structure that contains different pieces of
+information on the retrieved dataset, including file names of the
+file names on hard disk::
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: file names of the file names ?

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 5]
```diff
@@ -52,47 +55,52 @@ They return a structure that contains the different file names::
...
-
-  * default system paths used by third party softwares that may already have
-    downloaded of could benefit of this data
+Besides convenient downloading of openly accessible reference datasets
+including important meta-data (e.g., stimulus characteristics and
+participant information for confound removal), the fetching functions
+perform data downloads only once and return the locally saved data upon
+any later function calls.
+The locally stored data can be found in one of the
+following directories (in order of priority; if present):
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: pète un coup please !

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 6]
```diff
@@ -133,14 +143,16 @@ downloaded, a single line is needed to load it.
...
 
-    We can find two main representations for MRI scans:
+    It is important to appreciate two main representations for
+    storing and accessing more than one nifti images, that is sets
+    of MRI scans:
 
-    - a big 4D matrix representing 3D MRI along time, stored in a big 4D
-      NifTi file.
+    - a big 4D matrix representing (3D MRI + 1D for time), stored in a single
+      nifti file.
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: Nifti vs nifti doesn't seem to be consistent. I don't really know which one is preferable.

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 7]
```diff
@@ -149,22 +161,22 @@ downloaded, a single line is needed to load it.
...
-  * a file path to a Nifti or Analyse image
-  * any object exposing ``get_data()`` and ``get_affine()`` methods, for
-    instance a ``Nifti1Image`` from nibabel_.
+  * A string variable with a file path to a Nifti or Analyse image
+  * Any object exposing ``get_data()`` and ``get_affine()`` methods, typically
+    a ``Nifti1Image`` from nibabel_.
 
-**Niimg-4D:** Similarly, some functions require 4-dimensional Nifti-like
-data, which we call Niimgs, or Niimg-4D. Accepted inputs are then:
+**Niimg-4D:** Similarly, some functions require Nifti-like
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: why did you remove 4D this line is about 4D Niimg-like object ...

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 8]
```diff
@@ -222,14 +234,15 @@ can be easily extracted from the fMRI data using the
...
+--------------------------------------
 
 fMRI data is usually represented as a 4D block of data: 3 spatial
-dimensions and one of time. In practice, we are most often only
-interested in working only on the time-series of the voxels in the
-brain. It is thus convenient to apply a brain mask and go from a 4D
-array to a 2D array, `voxel` **x** `time`, as depicted below:
+dimensions and one time dimension. In practice, we are usually
+interested in working on the voxel time-series in the
+brain. It is thus convenient to apply a brain mask in order to translate the
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: translate is a bit ambiguous I find especially the mathematical meaning, maybe transform ?

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 9]
```diff
@@ -222,14 +234,15 @@ can be easily extracted from the fMRI data using the
...
+4D brain images representation into a compressed 2D data representation,
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: I don't really see what compressed means here.

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 10]
```diff
@@ -311,20 +334,24 @@ use file names as input parameters.
...
+Functional MRI data can be considered "high dimensional" given the
+p-versus-n ratio (e.g., p=~50,000-200,000 voxels for n=1000 samples).
+In this setting, machine-learning
+algorithms can perform poorly (i.e., curse-of-dimensionality problem).
+However, simple means from the realms of classical statistics can help
 reducing the number of voxels.
 
-The Student's t-test (:func:`scipy.stats.ttest_ind`) performs a simple statistical test that determines if two
+The Student's t-test (:func:`scipy.stats.ttest_ind`) is an established
+mean to determine whether two
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: mean -> means or maybe method, way, etc ...

File: doc/manipulating_visualizing/manipulating_images.rst [hunk 11]
```diff
@@ -357,12 +386,15 @@ is easy thanks to numpy indexing a la Matlab.
...
-We now want to restrict our study to the ventral temporal area. The
-corresponding mask is provided in `haxby.mask_vt`. We want to compute the
-intersection of this mask with our mask. The first step is to load it with
-nibabel's :func:`nibabel.load`. We then use a logical "and"
+We now want to restrict our investigation to the ventral temporal area. The
+corresponding spatial mask is provided in `haxby.mask_vt`.
+We want to compute the
+intersection of this provided mask with our self-computed mask.
+The first step is to load it with
+nibabel's :func:`nibabel.load`. We can then a logical "and" operation
```
Comment (lesteve) [inline doc/manipulating_visualizing/manipulating_images.rst]: missing verb: We can then *use*/*apply* a logical "and" operation

File: doc/manipulating_visualizing/manipulating_images.rst (hunk 8 above)
Comment (banilo) [inline doc/manipulating_visualizing/manipulating_images.rst]: I wanted to explicitly not use Transform() to paraphrase the function name intentionally.

**Comment by lesteve** [comment]
Good stuff, merging!

**Comment by GaelVaroquaux** [comment]
> Merged #619.

Good job!
